common/models/acl.js

```diff
@@ -515,8 +518,11 @@ module.exports = function(ACL) {
    * @param {Function} cb Callback function
    */
   ACL.resolvePrincipal = function(type, id, cb) {
```

Comment:
here in addition to the changes required by this PR, i'm turning ACL.resolvePrincipal() promisable with loopback utils